.clang-tidy

```diff
@@ -0,0 +1,12 @@
+Checks:
+  *,
+  -altera*,
+  -llvm*,
+  -fuchsia-default-arguments-*,
+  -bugprone-easily-swappable-parameters,
+  -cppcoreguidelines-avoid-magic-numbers,
+  -modernize-use-trailing-return-type,
```

Comment:
Nice.

### References
[1] https://releases.llvm.org/10.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/modernize-use-trailing-return-type.html#:~:text=Rewrites%20function%20signatures%20to%20use,parameter%20list%20(and%20qualifiers).

Reply:
Yeah trailing return type never really caught on lol.